app/services/protective-monitoring-service.js

```diff
@@ -0,0 +1,32 @@
+const { PublishEvent } = require('ffc-protective-monitoring')
+
+async function sendEvent (request, claim, event) {
+  const protectiveMonitoring = new PublishEvent(process.env.PROTECTIVE_MONITORING_URL)
```

Comment:
This value should be source from the config like the others not direct.  That way we can validate and better communicate the environment configuration in one place.